http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/LeoAppServiceInterp.scala

```diff
@@ -678,27 +655,10 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
           )
         else F.unit
 
-      // Save or retrieve a nodepool record for the app
-      userNodepoolOpt <- nodepoolQuery
-        .getMinimalByUserAndConfig(originatingUserEmail, cloudContext, machineConfig)
-        .transaction
-      clusterId = saveClusterResult.minimalCluster.id
-      nodepool <- userNodepoolOpt match {
-        case Some(n) =>
-          log.info(ctx.loggingCtx)(
-            s"Reusing user's nodepool ${n.id} in ${saveClusterResult.minimalCluster.cloudContext.asStringWithProvider} with ${machineConfig}"
-          ) >> F.pure(n)
-        case None =>
-          for {
-            _ <- log.info(ctx.loggingCtx)(
-              s"No nodepool with ${machineConfig} found for this user in project ${saveClusterResult.minimalCluster.cloudContext.asStringWithProvider}. Will create a new nodepool."
-            )
-            saveNodepool <- F.fromEither(
-              getUserNodepool(clusterId, cloudContext, originatingUserEmail, machineConfig, ctx.now)
-            )
-            savedNodepool <- nodepoolQuery.saveForCluster(saveNodepool).transaction
-          } yield savedNodepool
-      }
+      // Use the default nodepool of the cluster.
```

Comment:
Before, this code was using the v1 pattern where we create a new nodepool record for each unique user in a cluster. This was causing performance issues when there are many users in the same cluster, since each createApp request will query _all_ nodepools in the cluster. :( Moreover these nodepool records were unused for Azure apps, and did not get deleted when the app is deleted.

So I am just creating apps in the "default" nodepool of the cluster. Longer term, when we shift to v2 for GCP, we should revisit our DB schema and probably get rid of KUBERNETES_CLUSTER and NODEPOOL tables.